pkg/vcdcsiclient/common_system_test.go

```diff
@@ -17,6 +17,20 @@ var (
 	gitRoot string = ""
 )
 
+// NOTE: The test util functions in vcdsdk/common_system_test.go are being replicated in this file
+// because it is not possible to import functions or variables declared in _test.go files belonging to
+// a different package
+
+type authorizationDetails struct {
```

Comment:
Can we keep the type with leading caps (`AuthorizationDetails`)? I understand that we don't want to export but lower-case type seems odd. I need to check the go doc for this. Does GoLand complain about it?